data/policies/let.cas

```diff
@@ -8,8 +8,7 @@ let rw_file_perms = [ read_file_perms write ];
 
 resource bar {}
 
-// TODO: re-add once global binding is using Contexts
-//let baz = bar;
+let baz = bar;
```

Comment:
I am guessing bar has to actually exist here.  Maybe worthwhile adding an error policy to check that.